source/cloud/terraform.md

```diff
@@ -134,24 +136,20 @@ The file now contains the definition for one VM with several options you can cus
 | image_name    | Sets the operating system image for the machine. | See [Image list](https://cloud.vscentrum.be/dashboard/project/images)
 | flavor_name   | Sets the machine flavor. | see [Flavors list](flavors.md).
 | nginx_enabled | Installs nginx and exposes port 80. | true, false
-| nfs_enabled   | Connects the vfm to the NFS network. Only set true if you requested access. | true, false
-| vsc_enabled   | Connects the vfm to the VSC network. Only set true if you requested access. | true, false |
+| nfs_network   | Connects the vm to the NFS network. Only set true if you requested access  | true, false |
+| nfs_enabled   | DEPRECATED, Use nfs_network variable and nfs_share module instead | true, false |
+| vsc_enabled   | Connects the vm to the VSC network. Only set true if you requested access. | true, false |
 | rootdisk_size | Manually sets the size of the rootdisk, overriding the flavor settings | (number)
-More advanced options are described further on. |
 | is_windows | Configures windows-specific behavior if `true` | true/false |
-### Cluster of VMs
-You can also deploy a public VM and multiple private VMs (without a public IP) in one go by using the cluster template:
-```shell 
-cat cluster.tf.example >> main.tf
-```
-The customization options are similar to the single vm template:
-| Variable | Explanation
-|--|--|
-| private count | How many private instances you want |
-| public_flavor | The flavor for the public instances (and private instances by default)|
-| public_image | The OS image for the public instances (and private instances by default)|
-| cluster_name | The basename for the vms. Public instance will be called `MyCluster-public` and private instances `MyCluster-private-x` |
 
+More advanced options are described further on. 
+### Private VMs
+You can also deploy a public VM and multiple private VMs (without a public IP) by defining multiple instance modules and setting `public = false` on the private VMs.
+You can then reach those VMs through your public VMs with ssh-agent forwarding (`ssh -A`). See `frontend_backend_nfs.example` for a complete example.
+
+:::{warning}
+Automated features like auto-mounting volumes will **not** work on private VMs and are automatically disabled.
```

Comment:
This could be confusing for users, at least at this point in the doc, maybe it should be just a note or included later it